### DISCUSSION TITLE
Anywhere to download a Portable mode zip file from latest build?

### DISCUSSION THREAD

**Discussion by sharpninja** [microsoft/vscode-discussions#18] [Extension Development QnA] [answered]
I need a portable mode archive that has already had portable mode turned on.  Is there somewhere that keeps a latest build created?

**Comment by alefragnani** [comment, marked as answer]
Hi @sharpninja ,

There is no **official** location for that. 

If fact, there is no Portable Mode version/build, but a _step by step procedure_ to make the **.zip download** to work as Portable Mode. 

More details here https://code.visualstudio.com/docs/editor/portable#_enable-portable-mode

**Comment by sharpninja** [comment]
That requires being able to download and install normally, which defeats the purpose of portable mode.  I was hoping there was an official portable mode archive.

**Reply by alefragnani** [reply]
Nop...no installation is required. 

You just download the zip file, _unzip_ it and follow the instruction about the `data` folder, depending of the OS you use.

**Reply by digitarald** [reply]
https://code.visualstudio.com/download has zip files for all OSs.
